client/App.vue

```diff
@@ -173,4 +177,33 @@ input {
   margin: 0.5rem;
 }
 
+.alerts {
+  position: absolute;
+  z-index: 99;
+  bottom: 0;
```

Comment:
the alerts block is slightly below my window (see below screenshot, had to scroll a bit to see it). maybe we can change bottom/top positioning? i might actually suggest flashing alerts from the top of the page instead of the bottom, because it's easier to position overall.